### PR TITLE
Adding Article and Video Content

### PR DESCRIPTION
**Pull request by lmedury**
This PR is part of the EvmosNation grant project. In this PR, the following articles are included:

1. Project Spotlight: Orbit Marketplace
2. Project Spotlight: Stride
3. Project Spotlight: Tashi
4. React Web-App for dApp Interaction
5. Project Spotlight: Forge
6. Building a Payroll Smart Contract in Solidity and Deploying it to Evmos Testnet

And, the following video is also included:

1. Connect to Evmos Testnet using MetaMask
2. Connect to Evmos Testnet using Keplr Wallet
3. Submit Transactions using MetaMask Wallet
4. Create a dApp on Evmos Testnet

### CODE REVIEW

File: docs/articles/beginner/Learn/stride.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+![Stride Liquid Staking](/img/articles/Stride-1.png)
+
+### Incentivized Staking and Testnet Users
+
+Chains participating in Stride's liquid staking join the Airdrop Program, where addresses staking the chain’s governance token are eligible for a share of airdropped STRD based on their stakes. A minimum threshold of $100 and a maximum threshold of $50,000 are in place, ensuring effective management of addresses during the distribution process. Addresses below the minimum or exceeding the maximum threshold are ineligible or limited, respectively.
+
+![Stride Incentivised Staking](/img/articles/Stride-2.png)
+
+### Providing Liquidity on Osmosis
```
Comment (danburck) [inline docs/articles/beginner/Learn/stride.md:33]: This section should rather be about providing liquidity on Evmos, e.g. Forge

File: docs/articles/beginner/Learn/react.md [hunk 2]
```diff
@@ -0,0 +1,341 @@
+# React Web-App for dApp Interaction
+
+In the previous tutorial, we developed a smart contract using Remix IDE and deployed it on Evmos Testnet using Metamask wallet. Completing the previous tutorial before moving on to this one is strongly recommended.
```
Comment (danburck) [inline docs/articles/beginner/Learn/react.md:3]: No reference to a previous tutorial

File: docs/articles/beginner/Learn/react.md [hunk 3]
```diff
@@ -0,0 +1,341 @@
+# React Web-App for dApp Interaction
```
Comment (danburck) [inline docs/articles/beginner/Learn/react.md:1]: filename is generic, be more descriptive

File: docs/articles/beginner/Learn/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
+# Building a Payroll Smart Contract in Solidity and Deploying it to Evmos Testnet
```
Comment (danburck) [inline docs/articles/beginner/Learn/tutorial.md:1]: filename is generic, be more descriptive

**Review by danburck — COMMENTED** [review]
Please adjust the comments

File: docs/articles/beginner/Learn/stride.md (hunk 1 above)
Comment (lmedury) [reply docs/articles/beginner/Learn/stride.md]: Removed this section as we now have a separate article on Forge.

File: docs/articles/beginner/Learn/react.md (hunk 2 above)
Comment (lmedury) [reply docs/articles/beginner/Learn/react.md]: Added a reference

File: docs/articles/beginner/Learn/react.md (hunk 3 above)
Comment (lmedury) [reply docs/articles/beginner/Learn/react.md]: Renamed the file

File: docs/articles/beginner/Learn/tutorial.md (hunk 4 above)
Comment (lmedury) [reply docs/articles/beginner/Learn/tutorial.md]: Renamed the file

**Review by danburck — APPROVED** [review]

**Pull request merged**